client/ayon_blender/api/pipeline.py

```diff
@@ -460,25 +469,27 @@ def _discover_gui() -> Optional[Callable]:
     return None
 
 
-def add_to_avalon_container(container: bpy.types.Collection):
-    """Add the container to the Avalon container."""
+def add_to_ayon_container(container: bpy.types.Collection):
+    """Add the container to the Ayon container."""
 
-    avalon_container = bpy.data.collections.get(AVALON_CONTAINERS)
-    if not avalon_container:
-        avalon_container = bpy.data.collections.new(name=AVALON_CONTAINERS)
+    ayon_container = bpy.data.collections.get(AVALON_CONTAINERS) or (
```

Comment:
```suggestion
    # Prioritize the legacy name here so that in existing scenes, if user would downgrade
    # to an older release the new data would also be in the legacy container.
    ayon_container = bpy.data.collections.get(AVALON_CONTAINERS) or (
```